di_website/visualisation/static/visualisation/js/chart.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+const initPlotlyChart = () => {
```

Comment:
@edwinmp 

Latest JS code is here, but is probably going to change as we progress - feel free to take a copy for now though: 
https://github.com/devinit/DIwebsite-redesign/blob/feature/visualisation_june_2020/patterns/assets/js/charts/entry.js